includes/admin/class-wp-job-manager-admin.php

```diff
@@ -166,10 +166,14 @@ public function admin_enqueue_scripts() {
 	 * Adds pages to admin menu.
 	 */
 	public function admin_menu() {
+		remove_submenu_page( 'edit.php?post_type=job_listing', 'post-new.php?post_type=job_listing' );
+		$item = remove_submenu_page( 'edit.php?post_type=job_listing', 'edit.php?post_type=job_listing' );
+		// change item label to "Job Listings".
+		add_submenu_page( 'edit.php?post_type=job_listing', $item[0], __( 'Job Listings', 'wp-job-manager' ), $item[1], $item[2], '', 0 );
 		add_submenu_page( 'edit.php?post_type=job_listing', __( 'Settings', 'wp-job-manager' ), __( 'Settings', 'wp-job-manager' ), 'manage_options', 'job-manager-settings', [ $this->settings_page, 'output' ] );
 
 		if ( WP_Job_Manager_Helper::instance()->has_licensed_products() || apply_filters( 'job_manager_show_addons_page', true ) ) {
-			add_submenu_page( 'edit.php?post_type=job_listing', __( 'WP Job Manager Add-ons', 'wp-job-manager' ), __( 'Add-ons', 'wp-job-manager' ), 'manage_options', 'job-manager-addons', [ $this, 'addons_page' ] );
+			add_submenu_page( 'edit.php?post_type=job_listing', __( 'WP Job Manager Extensions', 'wp-job-manager' ), __( 'Extensions', 'wp-job-manager' ), 'manage_options', 'job-manager-addons', [ $this, 'addons_page' ] );
```

Comment:
I think the latest is that we are calling this 'Marketplace': p6r3EZ-1Nu-p2#comment-3379

Reply:
Good catch! Renamed in a79e326 and 941dfb8